tests/clients/test_heatlhcheck.py

```diff
@@ -62,7 +62,7 @@ def test_wait_until_ready_success_with_api_token(self, mock_urlopen):
         self.assertIn('Dapr-api-token', headers)
         self.assertEqual(headers['Dapr-api-token'], 'mytoken')
 
-    @patch.object(settings, 'DAPR_HEALTH_TIMEOUT', 2)
+    @patch.object(settings, 'DAPR_HEALTH_TIMEOUT', '2')
```

Comment:
Let's make this 2.5, so it's more explicit.
```suggestion
    @patch.object(settings, 'DAPR_HEALTH_TIMEOUT', '2.5')
```

Reply:
updated in [this commit](https://github.com/dapr/python-sdk/pull/692/commits/7903e88712effe4f4d893b95a47182345d9f1373)